### DISCUSSION TITLE
A simple idea for a single icon

### DISCUSSION THREAD

**Discussion by DendyGG** [JordanSantiagoYT/FNF-JS-Engine#817] [Ideas]
In Healthicon.hx under the icon code add

```
if (width == 150) {
loadGraphic(file, true, Math.floor(width), Math.floor(height)); //single icons on top
iconOffsets[0] = (width - 150);
}
```
Under the updateHitbox() function, add

```
if (width2 == 150) {
animation.add(char, [0], 0, false, isPlayer);
}
```

thus, you can add an icon image with a size of 150x150

for example
![icon-strident](https://github.com/user-attachments/assets/b05636d6-d963-437e-8033-331029751a75)

result 
![stridentbamba](https://github.com/user-attachments/assets/72400c17-f21f-43c8-94ab-7c2d7b09c908)

**Comment by TheStinkern** [comment]
just paste the icon on the losing icon
